Gemfile

```diff
@@ -21,7 +21,6 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
 end
 
-gem 'acts_as_tenant'
```

Comment:
If `acts_as_tenant` is fully removed from the `Gemfile` how can it be required in the below files? Is it pulled in elsewhere?

Reply:
@syncrou Its defined in the managed-api-common gem here
  https://github.com/ManageIQ/manageiq-api-common/blob/a674fe29752de460977b445a482fcc2a8db728c7/manageiq-api-common.gemspec#L16